docs/it/entity_configuration.rst

```diff
@@ -13,65 +13,58 @@ Un **Entity Configuration (EC)** è un metadata di federazione in formato Jose e
 Firma di Configuration
 ++++++++++++++++++++++
 
-La firma dei JWT :rfc:`7515` avviene mediante l'algoritmo RSA SHA-256 (RS256). Tutti i partecipanti della Federazione DEVONO supportare questo algoritmo di firma. Tutte le operazioni di firma relative agli ES, EC e TM sono eseguite con le chiavi pubbliche di Federazione (distinguiamo le chiavi di Federazione da quelle di OIDC Core. Questi ultimi risiedono nei metadata OIDC. Un ES o EC contiene sia le chiavi pubbliche di Federazione che i metadata OIDC).
+La firma dei JWT :rfc:`7515` avviene mediante l’algoritmo RSA SHA-256 (RS256). Tutti i partecipanti della Federazione DEVONO supportare questo algoritmo di firma. Tutte le operazioni di firma relative agli ES, EC e TM sono eseguite con le chiavi pubbliche di Federazione (distinguiamo le chiavi di Federazione da quelle di OIDC Core. Questi ultimi risiedono nei metadata OIDC. Un ES o EC contiene sia le chiavi pubbliche di Federazione che i metadata OIDC).
 
 
 Metadata di Federazione
 +++++++++++++++++++++++
 
-OIDC Federation definisce i metadata di federazione contenenti le informazioni di seguito definite, e i metadata OIDC per 
-ogni tipo di entità.
+OIDC Federation definisce i metadata di federazione contenenti le informazioni di seguito indicate 
+e i metadata OIDC per ogni tipo di entità.
 
 
 Entity Configuration comuni
 +++++++++++++++++++++++++++
 
 .. list-table::
-   :widths: 20 20 40 20
+   :widths: 20 60 20
    :header-rows: 1
 
    * - **Claim**
-     - **Tipo**
      - **Descrizione**
      - **Obbligatorio**
    * - **iss**
-     - String
-     - Identificativo dell'entità che lo emette. 
+     - String. Identificativo dell’entità che lo emette. 
      - |check-icon|
    * - **sub**
-     - String
-     - Identificativo del soggetto a cui è riferito. 
+     - String. Identificativo del soggetto a cui è riferito. 
      - |check-icon|
    * - **iat**
-     - UNIX Timestamp
-     - Data di emissione. 
+     - UNIX Timestamp. Sata di emissione. 
      - |check-icon|
    * - **exp**
-     - UNIX Timestamp
-     - Data di scadenza.
+     - UNIX Timestamp. Data di scadenza.
```

Comment:
- UNIX Timestamp con l’istante di scadenza del JWT, codificato come NumericDate come indicato in
       `RFC7519 - JSON Web Token (JWT) <https://www.rfc-editor.org/rfc/rfc7519.html>`_